public/test.html

```diff
@@ -7,6 +7,35 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, viewport-fit=cover" />
   <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
   <title>Organic Maps Links Test Page</title>
+  <style>
+    @font-face { font-family: Roboto; src: url('/fonts/Roboto-Light.ttf'); }
```

Comment:
The size is not so critical, but is it really necessary? We like lightweight code )

Reply:
👍 changed to
```css
      font-family: sans-serif;
      font-weight: lighter;
```